src/user/models/kyc/kyc.service.ts

```diff
@@ -137,9 +138,12 @@ export class KycService {
     if (!user) throw new NotFoundException('DFX user not found');
     if (!KycCompleted(user.userData.kycStatus)) throw new ConflictException('KYC required');
 
+    const apiKey = this.walletService.getApiKeyInternal(wallet.name);
+    if (!apiKey) throw new ConflictException(`ApiKey for wallet ${wallet.name} not available`);
```

Comment:
![image](https://user-images.githubusercontent.com/85513542/196709212-d0d01206-43fc-4911-a376-a29110a1cecb.png)
=> it's probably rather an InternalServerError